src/libraries/System.Security.Cryptography.Cng/tests/PropertyTests.cs

```diff
@@ -160,5 +160,33 @@ public static void NullValueRoundtrip()
             CngProperty property = new CngProperty("banana", null, CngPropertyOptions.CustomProperty);
             Assert.Null(property.GetValue());
         }
+
+        [Fact]
+        public static void TestConstructorSpan()
+        {
+            var name = "dotnet-test";
+            ReadOnlySpan<byte> value = new byte[12];
+            value[5] = 1;
+            value[6] = 2;
+            value[7] = 3;
+
+            _ = new CngProperty(name, value, CngPropertyOptions.CustomProperty);
```

Comment:
Consider calling `GetValue` on the new property and asserting its value.
```suggestion
            CngProperty property = new CngProperty(name, value, CngPropertyOptions.CustomProperty);
            AssertExtensions.SequenceEqual(value, property.GetValue());
```